### DISCUSSION TITLE
Correcting spelling errors - Spanish

### DISCUSSION THREAD

**Discussion by espettia** [sumatrapdfreader/sumatrapdf#2919] [General]
Hello, I'm a spanish speaker user and I would like to correct some spelling errors and weird translations. Where can I find the files I can edit to change some messages in my language only? Are these kind of edits accepted?

**Comment by GitHubRulesOK** [comment]
see https://www.sumatrapdfreader.org/docs/Contribute-translation

**Reply by espettia** [reply]
what are the & symbols for?
![image](https://user-images.githubusercontent.com/75216702/188052043-ac28cb6a-149b-472c-bf73-1461e32a3143.png)

**Reply by GitHubRulesOK** [reply]
& means that  the following character is the hot key so in each section it must not be duplicated by an upper or lowercase character, fairly tricky not to use the first same letter so this is a poor selection section several P's thus
 `&Página siguiente` perhaps should be `Pági&na siguiente`    OR  `Página &siguiente`   `Página &anterior`(it does not have to be n or p) however beware that &A is used by &Atrás so there would then be a conflict when active

![image](https://user-images.githubusercontent.com/12754913/188153376-519dc48f-4291-4673-9c7e-6c8dcc7e3400.png)
